src/main/java/io/ddavison/conductor/Locomotive.java

```diff
@@ -124,14 +107,7 @@ public Locomotive() {
             case CHROME:
                 capabilities = DesiredCapabilities.chrome();
                 if (isLocal) try {
-
-                    // Heroku Check
-                    String herokuChromeDriver = JvmUtil.getJvmProperty("GOOGLE_CHROME_BIN");
-                    System.out.println("GOOGLE_CHROME_BIN: " + herokuChromeDriver);
-
-                    if (herokuChromeDriver != null && !herokuChromeDriver.isEmpty()) {
-                        System.setProperty("webdriver.chrome.driver", herokuChromeDriver);
-                    }
+                    ChromeDriverManager.getInstance().setup();
 
                     ChromeDriverService service = new ChromeDriverService.Builder()
                             .usingDriverExecutable(new File(JvmUtil.getJvmProperty("webdriver.chrome.driver")))
```

Comment:
It shouldn't affect anything but this `builder` stuff can go too, it was also part of the heroku thing.